app/forms/mappings/Constraints.scala

```diff
@@ -56,6 +56,12 @@ trait Constraints {
         }
     }
 
+  protected def maxValue(maximum: Int, errorKey: String, input: String) =
+    input.toInt match {
+      case x if x < maximum => Invalid(errorKey, maximum)
```

Comment:
Invalid if it's less than the maximum? Looks a bit strange to me but I might be missing something 😄

Reply:
not in use-deleted as using regex not this code